### ISSUE TITLE
Updated from 2020.3 -> 2020.4 beta 3, Settings is asking me to update the app

### ISSUE THREAD

**Issue by OnlinePlayer865** [mullvad/mullvadvpn-app#1697]
## Issue report
App is telling me to update even though I am on the latest, 2020.4-beta3.

Operating system: Windows 10. 1909 (OS Build 18363.778)

App version: 2020.4-beta3

### Issue description
Just updated from 2020.3 -> 2020.4-beta3. Installation went good. But when I went into the app and into Settings I see a red exclamation mark next to "App Version". It says 2020.4-beta3, but below that box it then says "Update available, download to remain safe". I just installed 2020.4-beta3. I even quit the app and re-installed it using the installer that I downloaded from the Releases tab. I get the same issue in Settings.

**Comment by rr0ss0rr** [comment]
MacOS .. Looks like Beta 3 is reporting Beta 2
```
Current version: 2020.4-beta2
	Is supported: true
	Is up to date: false
Latest version: 2020.4-beta3 (latest stable: 2020.3)```

**Comment by dlon** [comment]
Thanks for mentioning this issue. There is a fix in master now which should be included in the next release.

**Comment by OnlinePlayer865** [comment]
Than you for the fix!

**Comment by faern** [comment]
Fixed by #1699
